kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -287,10 +287,10 @@ class KyuubiSessionImpl(
   }
 
   @volatile private var engineLastAlive: Long = _
-  val engineAliveTimeout = sessionConf.get(KyuubiConf.ENGINE_ALIVE_TIMEOUT)
-  val aliveProbeEnabled = sessionConf.get(KyuubiConf.ENGINE_ALIVE_PROBE_ENABLED)
-  var engineAliveMaxFailCount = 3
-  var engineAliveFailCount = 0
+  private val engineAliveTimeout = sessionConf.get(KyuubiConf.ENGINE_ALIVE_TIMEOUT)
+  private val aliveProbeEnabled = sessionConf.get(KyuubiConf.ENGINE_ALIVE_PROBE_ENABLED)
+  private val engineAliveMaxFailCount = 3
```

Comment:
the original PR does not contain such change

Reply:
This includes some of the changes in #5244. If we are not going to backport #5244, I think this change would be better.

Reply:
I noticed #5251 was planned to be merged into 1.8 in the comments, but it wasn't, do we need to merge it into 1.8 first? cc @cxzl25

Reply:
Sorry, my mistake may be that I picked branch 1.8 and did not push it.

Reply:
OK, I'll send a PR to backport it later.